.ko.yaml

```diff
@@ -1 +1,31 @@
-defaultBaseImage: gcr.io/k8s-prow/git:v20211018-372b1c8
+# Distroless images:
+# - prow/cmd/admission
+# - prow/cmd/gerrit
+# - prow/cmd/mkpj
+# - prow/cmd/mkpod
+# - prow/cmd/pipeline
+# defaultBaseImage: gcr.io/distroless/static:nonroot
```

Comment:
Did you mean for this to be uncommented?

Reply:
doesn't have to, this is the implicit default, I'm adding a comment here to make it a bit more obvious to code reviewers

Reply:
FWIW the explicit default is better in my opinion. Implicit defaults are invisible and could maybe change. Doesn't matter much though.